src/com/qamp/HarisJasarevic/homeworks/homework5/task1/SavingsAccount.java

```diff
@@ -0,0 +1,54 @@
+package com.qamp.HarisJasarevic.homeworks.homework5.task1;
+
+public class SavingsAccount extends BankAccounts {
+
+    private double dailyTransactionLimit;
+    private int transactionsLeft;
+
+    protected SavingsAccount (final AccountOwner accountOwner, final String serialNumber, final int passCode, final int dailyTransactionLimit) {
+        super(accountOwner, serialNumber, passCode);
+        this.dailyTransactionLimit = 1000;
```

Comment:
it should be possible to set the transaction limit via constructor, and if not it should be 1000, you can even use 2 constructors